src/test/java/de/uhd/ifi/se/decision/management/jira/model/decisionknowledgeelement/TestDecisionKnowledgeElementStrings.java

```diff
@@ -3,137 +3,172 @@
+    }
+
+    @Test
+    public void testSetId() {
+	this.decisionKnowledgeElement.setId(this.id + 1);
+	assertEquals(this.id + 1, this.decisionKnowledgeElement.getId());
+    }
+
+    @Test
+    public void testSetSummary() {
+	this.decisionKnowledgeElement.setSummary(this.summary + "New");
+	assertEquals(this.summary + "New", this.decisionKnowledgeElement.getSummary());
+    }
+
+    @Test
+    public void testSetDescription() {
+	this.decisionKnowledgeElement.setDescription(this.description + "New");
+	assertEquals(this.description + "New", this.decisionKnowledgeElement.getDescription());
+    }
+
+    @Test
+    public void testSetType() {
+	this.decisionKnowledgeElement.setType(KnowledgeType.ALTERNATIVE);
+	assertEquals(KnowledgeType.ALTERNATIVE, this.decisionKnowledgeElement.getType());
+    }
+
+    @Test
+    public void testSetProjectKey() {
+	this.decisionKnowledgeElement.setProject(this.projectKey + "New");
+	assertEquals(this.projectKey + "New", this.decisionKnowledgeElement.getProject().getProjectKey());
+    }
+
+    @Test
+    public void testGetKeyKeyNull() {
+	KnowledgeElement decisionKnowledgeElement = new KnowledgeElementImpl();
+	decisionKnowledgeElement.setProject("TEST");
+	decisionKnowledgeElement.setId(10);
+	assertEquals("TEST-10", decisionKnowledgeElement.getKey());
+    }
+
+    @Test
+    public void testDecisionKnowledgeElementInitialized() {
+	assertNotNull(this.decisionKnowledgeElement);
+    }
+
+    @Test
+    public void testEqualsNotTheObject() {
+	KnowledgeElement element = new KnowledgeElementImpl();
+	element.setId(123);
+	assertFalse(this.decisionKnowledgeElement.equals(element));
+    }
+
+    @Test
+    public void testGetDocumentationLocationAsStringNull() {
+	this.decisionKnowledgeElement.setDocumentationLocation((DocumentationLocation) null);
+	assertEquals("", decisionKnowledgeElement.getDocumentationLocationAsString());
+    }
+
+    @Test
+    public void isLinked() {
+	assertEquals(0, decisionKnowledgeElement.isLinked());
+    }
+
+    @Test
+    public void testGetDecisionGroups() {
+	List<String> groups = decisionKnowledgeElement.getDecisionGroups();
+	assertEquals("TestGroup", groups.get(0));
+    }
+
+    @Test
+    public void testAddDecisionGroupSingle() {
+	String group = "NewTestGroup";
+	decisionKnowledgeElement.addDecisionGroup(group);
+	assertTrue(decisionKnowledgeElement.getDecisionGroups().contains("NewTestGroup"));
+    }
+
+    @Test
+    public void testAddDecisionGroupsList() {
+	List<String> groups = new ArrayList<String>();
+	groups.add("ListTestGroup");
+	decisionKnowledgeElement.addDecisionGroups(groups);
+	assertTrue(decisionKnowledgeElement.getDecisionGroups().contains("ListTestGroup"));
+    }
+
+    @Test
+    public void testRemoveDecisionGroup() {
+	decisionKnowledgeElement.removeDecisionGroup("TestGroup");
+	assertTrue(!decisionKnowledgeElement.getDecisionGroups().contains("TestGroup"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [assertTrue(!expr) can be replaced by assertFalse(expr)](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)